pkgs/generate-secrets/generate-secrets.sh

```diff
@@ -11,10 +11,12 @@ makePasswordSecret lnd-wallet-password
 makePasswordSecret liquid-rpcpassword
 makePasswordSecret lightning-charge-token
 makePasswordSecret spark-wallet-password
+makePasswordSecret backup-encryption-password
 
 [[ -e lightning-charge-env ]] || echo "API_TOKEN=$(cat lightning-charge-token)" > lightning-charge-env
 [[ -e nanopos-env          ]] || echo "CHARGE_TOKEN=$(cat lightning-charge-token)" > nanopos-env
 [[ -e spark-wallet-login   ]] || echo "login=spark-wallet:$(cat spark-wallet-password)" > spark-wallet-login
+[[ -e backup-encryption-env ]] || echo "PASSPHRASE=$(cat backup-encryption-password)" > backup-encryption-env
```

Comment:
This is not safe as any user running `ps aux | grep 'echo PASSPHRASE` in the right moment (can be achieved using infinite loop) will see the passphrase. Proper way to do it:

```bash
echo -n PASSPHRASE= | cat - backup-encryption-password > backup-encryption-env
```

Of course, the other lines seem to be vulnerable too. FYI, I'm not even sure if passing the passphrase via environment is safe - I've read that it isn't but I couldn't figure out how to abuse it.

Reply:
`echo` is a bash builtin, so its arguments are invisible outside of the script process.

Reply:
Would it make sense to write `builtin echo` in order to make double sure nothing screws up? I'm not sure how much is bash reliable in calling builtins.

Reply:
`echo` has been a builtin since the first Bash release in the late 80s. It's totally safe to use. :wink: